src/components/forms/pool_actions/InvestForm/composables/useInvestMath.ts

```diff
@@ -45,7 +47,7 @@ export default function useInvestFormMath(
    * COMPOSABLES
    */
   const { toFiat } = useNumbers();
-  const { tokens, balances, balanceFor, nativeAsset } = useTokens();
```

Comment:
This native asset returned from useTokens pulls the schema from the config. No need to import this manually above. https://github.com/balancer-labs/frontend-v2/blob/develop/src/providers/tokens.provider.ts#L112-L115

Reply:
Just pushed a fix for this.

The problem I was facing is that `nativeAsset` wasn't really a `TokenInfo`, it was more of a decorated one because it had custom props on it.

I've typed it now properly and it should be good now.